services/dy-static-file-server/docker/custom/Dockerfile

```diff
@@ -7,7 +7,7 @@ FROM joseluisq/static-web-server:2.0.2-alpine as production
 #     docker run dy-static-file-server:prod
 #
 
-ARG PYTHON_VERSION="3.8.10-r0"
+ARG PYTHON_VERSION="3.8.15-r0"
```

Comment:
you do not want to go straight for 3.11?
also the version of your web-server is trailing way behind. we use 2.32 in simcore.

Reply:
it depends on the version inside the base image of stati-web-server. So for now I just did a minor update